docker/README.md

```diff
@@ -1,3 +1,7 @@
-# Docker
+# Description
+This directory contains Docker files for building different flavors of Docker images:
+- *develop* contains Dockerfile with all the dependencies required for compiling Iroha source code. We use it in development process, thus all depencencies are kept up-to-date
+- *release* contains Dockerifle with all the dependencies required for running Iroha binaries compiled with *Release* flag. Each time commit is pushed into `develop` or `master` branch, automated routine is executed to build Docker image containing Iroha binaries. It is then pushed to [Dockerhub](https://hub.docker.com/r/hyperledger/iroha/)
```

Comment:
Doesn't everything, which goes after words "Each time commit is pushed..." relate to both *develop* and *release*? Or only *release* is updated, even if we push into 'develop'?
If does relate to both, please move this part to the beginning.